sql/core/src/test/scala/org/apache/spark/sql/DataFrameAggregateSuite.scala

```diff
@@ -727,4 +728,67 @@ class DataFrameAggregateSuite extends QueryTest with SharedSQLContext {
       "grouping expressions: [current_date(None)], value: [key: int, value: string], " +
         "type: GroupBy]"))
   }
+
+  def getEveryAggColumn(columnName: String): Column = {
+    Column(new EveryAgg(Column(columnName).expr).toAggregateExpression(false))
```

Comment:
Since we don't have APIs for them in `functions`, it's not likely users will use then with DataFrame. Thus I think we don't need these tests.

Reply:
@cloud-fan Ok, let me remove these.